README.md

```diff
@@ -1,284 +1,126 @@
 
-1. `cd` into the new directory (e.g. `cd my-app`)
+### 1. Create Next App
 
-1. `npm install -D @cloudflare/next-on-pages vercel`
+To start creating a Next.js application using next-on-pages, start by creating a standard Next.js application using the create-next-app command (skip this step if you already have an existing Next.js application that you want to port to Cloudflare using next-on-pages):
 
-1. Configure the project to use the Edge Runtime:
+```sh
+npx create-next-app@13.2.4 my-next-app
+```
 
-   Replace `pages/api/hello.ts` with the following:
+<details>
 
-   ```typescript
-   // Next.js Edge API Routes: https://nextjs.org/docs/api-routes/edge-api-routes
-   import type { NextRequest } from 'next/server';
+<summary>Note on the Next.js version</summary>
 
-   export const config = {
-   	runtime: 'experimental-edge',
-   };
+The above command suggests to use Next's `13.2.4` version, that is the latest at the time of writing, older versions are also supported (but they might be only partially), newer versions might not be so be wary of that. The team will soon test various versions and document more clearly which are supported to what degree.
 
-   export default async function handler(req: NextRequest) {
-   	return new Response(JSON.stringify({ name: 'John Doe' }), {
-   		status: 200,
-   		headers: {
-   			'Content-Type': 'application/json',
-   		},
-   	});
-   }
-   ```
+</details>
 
-1. `git commit` and `git push` to a GitHub/GitLab repository.
+&NewLine;
 
-1. Create a Pages project, connect that repository, and select "Next.js" from the framework preset list.
+Change your current directory to the newly created one, as the following step require you to be in it:
 
-   | Option                 | Value                                                 |
-   | ---------------------- | ----------------------------------------------------- |
-   | Build command          | `npx @cloudflare/next-on-pages --experimental-minify` |
-   | Build output directory | `.vercel/output/static`                               |
+```sh
+cd my-next-app
+```
 
-1. Add a `NODE_VERSION` environment variable set to `16` or greater (`18` is not supported yet, See [Build Image Update Discussion](https://github.com/cloudflare/pages-build-image/discussions/1).
+### 2. Configure the application to use the Edge Runtime
 
-1. In the Pages project **Settings** > **Functions** > **Compatibility Flags**, add the `nodejs_compat` and ensure the **Compatibility Date** is set to at least `2022-11-30`.
+In order for the application to run on Cloudflare Pages it needs to be set to run on the edge runtime, in order to do so make sure that all the files in your application containing server side code (as your api routes and pages using `getServerSideProps`) export a config object specifying the use of the edge runtime:
 
-1. The project should now be ready to deploy. Create a new deployment.
+```ts
+export const config = {
+	runtime: 'edge',
```

Comment:
This works fine for Edge API Routes, but I think will log a warning for an SSR pages which prompt users to continue using `experimental-edge`. This was true at least a little while ago. It might have now changed, rendering this comment moot. Feel free to disregard if I'm operating on outdated info.

Reply:
I think that `edge` is fine everywhere but if you don't provide it, our own warnings prompt you to use `experimental-edge` (instead of `edge`) and if you provide do `experimental-edge` you get a warning from the Vercel CLI.

I didn't want to add that change in this PR to keep the scope of this one to be just documentation, so I've updated our warning here instead: https://github.com/cloudflare/next-on-pages/pull/127